tensorboard/components/tf_dashboard_common/data-loader-behavior.ts

```diff
@@ -129,9 +129,9 @@ namespace tf_dashboard_common {
     },
 
     created() {
-      this._loadData = _.debounce(this._loadDataImpl, 100, {
+      this._loadData = _.throttle(this._loadDataImpl, 100, {
         leading: true,
-        trailing: true,
+        trailing: false,
```

Comment:
You want this to be true, too. When `dataToLoad` changes while `loadDataImpl` is doing its work, you want to invoke it again at the end so we fetch the new data we need to load. FWIW, I think `throttle` may make more sense than `debounce` so I like the change above :)